sw/device/rom_exts/rom_ext_start.S

```diff
@@ -92,8 +92,8 @@ _rom_ext_interrupt_vector:
   // On Ibex interrupt id 31 is for non-maskable interrupts.
   unimp
 
-  // Ibex Reset Handler:
-  j _rom_ext_start_boot
+  // ROM_EXT is the second boot stage, so we should never hit this entry.
+  unimp
```

Comment:
This instruction is intended to be the entry point of the ROM_EXT. If you look at the vector before, there are entries for 0-31, and this is entry idx 32, which starts `0x80` bytes after the start of the interrupt vector.

The ROM_EXT entry was designed to be broadly similar to how the Ibex entry/reset point works, on purpose.

Try this change, in addition to removing the `+4`:
```suggestion
  // ROM_EXT entry point.
  j _rom_ext_start_boot
```

Why do we start execution with a jump? Each entry in the interrupt vector is only one instruction, which means we can't jump to any arbitrary 32-bit offset, instead these entries can only do a relative jump with a 20-bit offset. 

To be safe, we ensure all functions that the vector might jump to are "close" to the jump instruction, which is what the `.crt` section is for. The idea is this will contain the handler functions (but not functions the handlers call). That should keep code size in this section down, so that the relative offsets in the jumps always fit into a single instruction.

But, we don't control the order of functions in the `.crt` section, so we pretend the entry address is just like any other kind of interrupt entry - and jump from it to a section in the `.crt`, rather than hoping that `_rom_ext_start_boot` is the first function in the `.crt` section and magically appears at the entry point.

Overall, the aim is to make the entry point code less brittle in the face of the linker rearranging all the handlers for these functions.

Reply:
I see what you have done there: `mask_rom_boot -> _rom_ext_interrupt_vector+32 -> _rom_ext_start_boot` and we know the address of `_rom_ext_interrupt_vector+32` deterministically. Makes sense, I actually missed the comment in `.hjson` that explains this. I will extend the comment to explain this.

Reply:
Done.